config/samples/httpd-overrides/README.md

```diff
@@ -0,0 +1,232 @@
+# Glance HTTPD Configuration Overrides
+
+The glance-operator provides mechanisms to customize the Apache HTTPD server
+configuration through the use of custom configuration files. This feature
+leverages the
+[ExtraMounts](https://github.com/openstack-k8s-operators/dev-docs/blob/main/extra_mounts.md)
+functionality to mount custom HTTPD configuration files into the Glance
+deployment.
+
+## How It Works
+
+1. **Custom Configuration Files**: Create HTTPD configuration files with your
+   custom settings
+2. **ConfigMap**: Create ConfigMaps from files containing the overrides
+3. **OpenStackControlPlane Patch**: Patch the control plane to mount the
+   generated ConfigMap into Glance containers. The HTTPD configuration
+   automatically includes files matching the naming convention
+   `conf/httpd_custom*`
+
+
+### Step 1: Create Custom HTTPD Configuration
+
+Create your custom HTTPD configuration file(s). The filename must start with
+`httpd_custom` to be automatically included by the base HTTPD configuration.
```

Comment:
wondering if it would be easier to just mount all the content of the httpd-overrides CM to an e.g.  `/etc/httpd/conf_custom` dir and just include whatever is in there?

Reply:
With this last change for a pure operator perspective I can also avoid to fix privileges through kolla on the existing directory, and the patch is reduced to:

```
IncludeOptional conf_custom/*.conf
```
within the vhost.